src/plugins/auto/plugin.cpp

```diff
@@ -453,6 +481,96 @@ QueryNetworkResult MultiDeviceInferencePlugin::QueryNetwork(const CNNNetwork&
     return queryResult;
 }
 
+void MultiDeviceInferencePlugin::SelectDeviceForMulti(const std::vector<DeviceInformation>& metaDevices,
```

Comment:
looks like these function is quite duplicated.
I was thinking of the change in exenetwork constructor, e.g, loadnetwork to MULTI in cumulative mode, the selectDevice may return the MULTI:deviceA,deviceB directly in cumulative, is this not applicable?